src/components/AccountWarning/CannedMessages.ts

```diff
@@ -294,7 +294,23 @@ moderator to let them know.
 Thanks for your recent report about {{bot}}.
 
 We've notified the owner of that bot.
-            `),
+`),
             { bot },
         ),
+    ack_suspended: (reported) =>
+        interpolate(
+            _(`
+Thank you for your report.  {{reported}} is a repeat offender, their account has been suspended.
+`),
+            { reported },
+        ),
+
+    ack_suspended_and_annul: (reported) =>
+        interpolate(
+            _(`
```

Comment:
Good candidates for `llm_pgettext` maybe?

Reply:
(It felt simple and stable to me, but hey why not :) )